src/pages/en/about/newcomer-guide.mdx

```diff
@@ -1,11 +1,10 @@
 ---
 title: Newcomers' Guide
-description: A newcomers'
+description: A newcomers' guide to Quilt—information, questions and concerns
```

Comment:
What's Quilt-information?

Reply:
I think it's meant to be Quilt - Information, questions, and answers

Reply:
Yeah it's an em dash, haha